packages/promotion/src/services/promotion-module.ts

```diff
@@ -71,6 +76,162 @@ export default class PromotionModuleService<
     return joinerConfig
   }
 
+  async computeActions(
+    promotionsToApply: Pick<PromotionTypes.PromotionDTO, "code">[],
```

Comment:
suggestion: Would it make sense to pass the codes as strings?
```suggestion
    promotionsToApply: string[],
```

Reply:
I thought the same but it seems that it is expecting an object including the code property, I am fine with just string[]